testData/ontologies/flat.json5

```diff
@@ -571,6 +571,13 @@
             "minimum": 0,
             "maximum": 1
         },
+        "http://registry.gpii.net/common/brightness": {
```

Comment:
Could we have this as "screenBrightness" just for a bit more clarity?

Reply:
Sure, doing it.